samples/NativeAOT/MainActivity.cs

```diff
@@ -10,6 +11,8 @@ public class MainActivity : Activity
 {
     protected override void OnCreate(Bundle? savedInstanceState)
     {
+        Log.Debug ("NativeAOT", "Application.OnCreate()");
```

Comment:
Debug message should say `MainActivity.OnCreate()`, not `Application.OnCreate()`.